deploy/architecture.md

```diff
@@ -1,8 +1,20 @@
-# Kubernetes deployment
+# DSS deployment architecture
```

Comment:
Is 'architecture' the correct concept here? In my mind in the context of DSS pooling as described by the introduction, an architecture would be describing a specific arrangement of DSSs forming a pool. But here it seems like we are talking about how to actually define such an architecture and how to operate it.